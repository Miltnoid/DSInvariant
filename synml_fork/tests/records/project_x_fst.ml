type nat =
  | O
  | S of nat

let getXfst : {x:(nat * nat)} -> nat |>
  { {x = (0,1)} => 0
  | {x = (1,0)} => 1 } = ?
