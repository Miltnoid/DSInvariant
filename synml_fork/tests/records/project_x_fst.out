(* Generated Tue Mar  3 15:20:27 2015 *)
let getXfst : { x : (nat * nat) } -> nat =
  fun (r1:{ x : (nat * nat) }) -> #1 (r1.x)
;;
